content/about.md

```diff
@@ -3,22 +3,28 @@ layout: page
 title: About
 ---
 
-**startword, _n._**
-
-> (computer science) A string of letters that identifies the beginning of a valid sequence in a specified language.
-
-> (literature) The first appearance of a genre in print.
-
-<p class="tc">❀</p>
-
-*Startwords* is a research periodical irregularly published by the Center for Digital Humanities at Princeton. More formal than a blog yet more speculative and process-based than a peer-reviewed journal, *Startwords* is a forum for experimental humanities scholarship. Embedded code, data physicalizations, and emerging forms of project documentation are explored through writing that is creative, conversational, exploratory, essayistic, data-driven, and academically rigorous.
-
-Each issue features two articles that are unified by a common theme, feature, or concern. Issues will be released one to three times a year, when we have something to share or say. Articles can be contributed by CDH staff, faculty, graduate students, or external collaborators.
+<dl>
```

Comment:
There is markdown syntax for definition lists, and it seems to be supported by whatever processor we're using.
Seems like it would be more readable to use markdown, and that we should prefer markdown to html where possible.
https://www.markdownguide.org/extended-syntax/#definition-lists